electron/src/lib/CoreProtocol.test.main.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from 'assert';
```

Comment:
Missing license header

Reply:
My favorite error 🗡. I should make more use of my IDE: https://www.jetbrains.com/help/webstorm/copyright.html